jdaviz/core/template_mixin.py

```diff
@@ -1433,6 +1460,10 @@ def __init__(self, plugin, items, selected, viewer,
         self._update_layer_items()
         self.update_wcs_only_filter(only_wcs_layers)
 
+        # TODO: all of these add_filter commands should be refactored to pass filters directly
```

Comment:
A tech debt JIRA ticket somewhere?

Reply:
not yet, but I have it in the description as a todo item (to create the tickets)